src/main/scala/com/gu/newsletterlistcleanse/GetCleanseListLambda.scala

```diff
@@ -1,10 +1,11 @@
 package com.gu.newsletterlistcleanse
 
+import java.io.InputStream
 import java.util.concurrent.TimeUnit
 
 import com.amazonaws.services.lambda.runtime.events.SQSEvent
 import com.amazonaws.services.sqs.model.SendMessageResult
-import com.gu.newsletterlistcleanse.db.{Campaigns, CampaignsFromDB}
+import com.gu.newsletterlistcleanse.db.{AthenaOperations, BigQueryOperations, DatabaseOperations}
```

Comment:
I think we can drop `AthenaOperations` here.
```suggestion
import com.gu.newsletterlistcleanse.db.{BigQueryOperations, DatabaseOperations}
```